src/App.css

```diff
@@ -435,3 +435,51 @@ input {
     --search-box-w: 90% !important;
   }
 }
+
+/* footer css */
+.main-footer {
+  padding-left: 4rem;
+  position: relative;
+  bottom: 0;
+  width: 100%;
+  text-align: center;
+  color: #15191d;
+  background-color: #fff;
+  float: left;
+  border-top: 3px solid aqua;
+  border-radius: 30%;
+}
+
+@media (max-width: 400px) {
+  .main-footer {
+    display: flex;
+    width: 100%;
+  }
+}
```

Comment:
You can use the already created media query instead ```@media (max-width: 480px){}```
![respo2](https://user-images.githubusercontent.com/60689731/82757991-8a847280-9dec-11ea-85ca-09a497a7fb33.png)
try using css grid instead, it's not yet responsive on smaller devices